js/interface.js

```diff
@@ -206,6 +206,11 @@ Fliplet.Widget.register('com.fliplet.theme', function() {
 
       Fliplet.App.Settings.set({ themeEngineVersion: '2.0.0' })
         .then(function() {
+          Fliplet.Analytics.trackEvent({
```

Comment:
Please confirm that `Fliplet.Analytics` works in widget interface and how to read the data if it does work.